config/settings.py

```diff
@@ -188,54 +192,32 @@
         },
         "celery": {"handlers": ["celery"], "level": "INFO", "propagate": True},
         "osidb": {"level": "WARNING", "handlers": ["console"], "propagate": False},
-        "collectors.bzimport": {
-            "level": "WARNING",
-            "handlers": ["console"],
-            "propagate": True,
-        },
-        "collectors.bzimport.tasks": {
-            "level": "WARNING",
-            "handlers": ["console"],
-            "propagate": True,
-        },
-        # TODO this is all repeated
-        # can it be defined just once ?
-        "collectors.errata": {
-            "level": "WARNING",
-            "handlers": ["console"],
-            "propagate": True,
-        },
-        "collectors.example": {
-            "level": "WARNING",
-            "handlers": ["console"],
-            "propagate": True,
-        },
-        "collectors.framework": {
-            "level": "WARNING",
-            "handlers": ["console"],
-            "propagate": True,
-        },
-        "collectors.jiraffe": {
-            "level": "WARNING",
-            "handlers": ["console"],
-            "propagate": True,
-        },
-        "collectors.jiraffe.tasks": {
-            "level": "WARNING",
-            "handlers": ["console"],
-            "propagate": True,
-        },
-        "collectors.product_definitions": {
-            "level": "WARNING",
-            "handlers": ["console"],
-            "propagate": True,
-        },
         "apps.osim": {
             "level": "WARNING",
             "handlers": ["console"],
             "propagate": True,
         },
         "django_auth_ldap": {"level": "WARNING", "handlers": ["console"]},
+        # Collectors loggers
+        **{
+            collector_name: {
+                "level": "WARNING",
+                "handlers": ["console"],
```

Comment:
shouldn't it use the `celery` handler?

Reply:
Yes it should, but it seems, that the `celery` logger handler overrides the handler for the collectors loggers anyway.